larray/core/array.py

```diff
@@ -5816,23 +5849,28 @@ def clip(self, a_min, a_max, out=None):
         from larray.core.ufuncs import clip
         return clip(self, a_min, a_max, out)
 
-    def to_csv(self, filepath, sep=',', na_rep='', transpose=True, dropna=None, dialect='default', **kwargs):
+    @deprecate_kwarg('transpose', 'wide')
+    def to_csv(self, filepath, sep=',', na_rep='', wide=True, value_name='value', dropna=None, dialect='default', **kwargs):
         """
         Writes array to a csv file.
 
         Parameters
         ----------
         filepath : str
             path where the csv file has to be written.
-        sep : str
-            seperator for the csv file.
-        na_rep : str
-            replace NA values with na_rep.
-        transpose : boolean
-            transpose = True  => transpose over last axis.
-            transpose = False => no transpose.
-        dialect : 'default' | 'classic'
-            Whether or not to write the last axis name (using '\' )
+        sep : str, optional
+            separator for the csv file. Defaults to `,`.
+        na_rep : str, optional
+            replace NA values with na_rep. Defaults to ''.
+        wide : boolean, optional
+            Whether or not writing arrays in "wide" format. If True, arrays are exported with the last axis
+            represented horizontally. If False, arrays are exported in "narrow" format: all axes and also the values
```

Comment:
"all axes and also the values are written as columns". This formulation is a bit weird => use something like "one column per axis plus one value column" instead?